src/Microsoft.Azure.SignalR/EndpointProvider/ServiceEndpointProvider.cs

```diff
@@ -22,7 +22,7 @@ internal class ServiceEndpointProvider : IServiceEndpointProvider
         private readonly TimeSpan _accessTokenLifetime;
         private readonly IServiceEndpointGenerator _generator;
 
-        public ServiceEndpointProvider(ServiceEndpoint endpoint, string appName = "", IOptions<ServiceOptions> serviceOptions = null, TimeSpan? ttl = null)
+        public ServiceEndpointProvider(ServiceEndpoint endpoint, IOptions<ServiceOptions> serviceOptions, TimeSpan? ttl = null)
         {
             var connectionString = endpoint.ConnectionString;

```

Comment:
for `serviceOptions`, check null

Reply:
But we alreadly check `connectionString` in next line (line 28)

Reply:
I think passing serviceOption is unnecessary, the provider uses ApplicationName only

Reply:
already update in commit a3513f7